### ISSUE TITLE
Chrome: Ck editor steals focus from other text feilds on the page

### ISSUE THREAD

**Issue by zoarif** [ckeditor/ckeditor-dev#555]
## Are you reporting a feature or a bug?

Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)
Found a really old bug:
http://ckeditor.com/forums/CKEditor/INLINE-EDITING-CKEditor-steals-focus

## Provide detailed reproduction steps (if any)

1. Add a ck editor on your page where you have some text fields. Focus on ck editor and start writing
2. Now click on some other text field.
3. The focus gets lost and stolen by ck editor. 

### Expected result

The text field should remain focused.

### Actual result

Focus gets stolen by CK

## Other details

* Browser: Chrome
* OS: Happening on all windows versions
* CKEditor version: 4.7.0
* Installed CKEditor plugins: …

**Comment by msamsel** [comment]
I could reproduce situation only in **inline** editor. You can use this snippet to check:
```
<!doctype html>
<html lang='en'>
	<head>
		<meta charset="utf-8">
		<title>TEST PAGE</title>
		<script src='../ckeditor.js'></script>
	</head>
	<body>
		<div id="editor1" contenteditable="true">
			<div>Test Test</div>
		</div>
		<textarea name="txt" id="txt" cols="30" rows="10"></textarea>

		<script>
			CKEDITOR.inline( 'editor1', {});
		</script>
	</body>
</html>
```
1. Click test
1. Start typing
1. Click **once** into `textarea` below.
1. There is no caret, so you cannot start to write there. It's required to click once again.

Note: In case of regular `div` with `contenteditable="true"` without loaded CKEditor, focus is given immediately.

**Comment by wojtekw92** [comment]
I'm not able to reproduce this issue on version 4.7.1.

**Comment by wojtekw92** [comment]
fixed in 4.7.1
